sidecar-injector/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.11-stretch as mutating-webhook-service-builder
+FROM golang:latest as mutating-webhook-service-builder
```

Comment:
We probably want to pin to an exact version of Go

Reply:
agreed.